apps/ios/Shared/Views/Chat/ChatItem/FramedItemView.swift

```diff
@@ -144,21 +190,17 @@ struct FramedItemView: View {
                 ciQuotedMsgView(qi)
             }
         }
-        .overlay(DetermineWidth())
-        .frame(minWidth: msgWidth, alignment: .leading)
-        .background(
-            chatItem.chatDir.sent
-            ? (colorScheme == .light ? sentQuoteColorLight : sentQuoteColorDark)
-            : Color(uiColor: .quaternarySystemFill)
-        )
-
+            .overlay(DetermineWidth())
+            .frame(minWidth: msgWidth, alignment: .leading)
+            .background(chatItemFrameContextColor(chatItem, colorScheme))
```

Comment:
what does this change does?

Reply:
same frame color is used for marked deleted section